components/BottomNavigation/src/MDCBottomNavigationBarController.h

```diff
@@ -0,0 +1,68 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class MDCBottomNavigationBar;
+@protocol MDCBottomNavigationBarDelegate;
+@protocol MDCBottomNavigationBarControllerDelegate;
+
+/**
+ * MDCBottomNavigationBarController is a class that manages the navigation bar that allows movement
+ * between primary destination in an app.  It ties a list of view controllers to the bottom
+ * navigation bar and will display the corresponding view controller in the content view when an
+ * in the navigation bar is selected.
+ */
+@interface MDCBottomNavigationBarController : UIViewController <MDCBottomNavigationBarDelegate>
+
+/**
+ * The bottom navigation bar that hosts the tab bar items.
+ * @warning This controller sets itself as the navigation bar's delegate.  If you would like to
+ * observe changes to the navigation bar, conform to \c MDCBottomNavigationBarControllerDelegate
+ * and set the delegate property of this controller.
+ */
+@property(nonatomic, strong, readonly) MDCBottomNavigationBar *navigationBar;
+
+/**
+ * An array of view controllers to display when their corresponding tab bar item is selected in the
+ * navigation bar.  When this property is set, the navigation bar's \c items property will be set to
+ * an array composed of the \c tabBarItem property of each view controller in this array.
+ * @see UIViewController#tabBarItem
+ */
+@property(nonatomic, copy) NSArray<UIViewController *> *viewControllers;
```

Comment:
Apple seems to be preferring `__kindof UIViewController *` in their APIs. Can we switch to the same declaration (throughout this file)?

Reply:
Done